README.md

```diff
@@ -2,13 +2,23 @@
 
 [![OSS Lifecycle](https://img.shields.io/osslifecycle/honeycombio/honeycomb-network-agent)](https://github.com/honeycombio/home/blob/main/honeycomb-oss-lifecycle-and-practices.md)
 
-The agent is deployed to Kubernetes as a [`DaemonSet`](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/),
-which means that Kubernetes will try to have the agent run on every node in the cluster.
+The Honeycomb Network Agent is a low effort, no-code, language agnostic solution to getting telemetry of your applications running in Kubernetes.
 
 Docker images are found in [`ghcr.io/honeycombio/network-agent:latest`](https://github.com/honeycombio/honeycomb-network-agent/pkgs/container/network-agent).
 
 See notes on local development in [`DEVELOPING.md`](./DEVELOPING.md)
 
+## How it Works
+
+The agent runs as a [`DaemonSet`](https://kubernetes.io/docs/admin/daemons/) on each node in a Kubernetes cluster.
```

Comment:
```suggestion
The agent runs as a [`DaemonSet`](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/) on each node in a Kubernetes cluster.
```
This link is broken and I couldn't find where it was supposed to land... any reason to not keep the original link we had here?

Reply:
Weird, not sure how I ended up with a bad link 🤷🏻